docs/content/users/quickstart.md

```diff
@@ -149,6 +149,40 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
         ddev launch
         ```
 
+=== "ExpressionEngine"
+
+    ## ExpressionEngine
+
+    === "ExpressionEngine Zipfile Download"
+
+        Download the ExpressionEngine code from [ExpressionEngine.com](https://expressionengine.com/), then:
+
+        ```bash
+        mkdir my-ee && cd my-ee
+        unzip /path/to/ee-zipfile.zip
+        ddev config # Accept the defaults
+        ddev start
+        ddev launch /admin.php
+        ```
+
+        In the `admin.php` setup, use "db" for the "DB Server Address", "DB Name", "DB Username", and "DB Password"
+
+        Visit your site.
+
+    === "ExpressionEngine Git Checkout"
+
+        ```bash
```

Comment:
Provide some context:

```suggestion
Follow the instructions in ExpressionEngine’s Git repository [README.md](https://github.com/ExpressionEngine/ExpressionEngine#how-to-install):

        ```bash
```

Reply:
I like the context. Mostly you can't just follow the instructions.

Reply:
I agree that the instructions here are much more straightforward, and it seems to me like they’re following the general steps anyway—the links out aren’t hard requirements (since the steps are immediately detailed below) but only providing context.

Reply:
I added the links, just with "based on"

Reply:
That reads better than my suggestion, thank you!